src/components/Link/Link.scss

```diff
@@ -49,3 +49,10 @@
 .ms-Link {
   @include ms-Link;
 }
+
+// For button links we need to ensure by default that we inherit the font size. Otherwise
+// the button default font size will override.
+
+BUTTON.ms-Link {
+  font-size: inherit;
```

Comment:
Interesting. Okay, I guess we need to remove `font-size` from `.ms-Link` to generally let it inherit, and *also* target `button.ms-Link` to override the browser default.